cfme/configure/access_control.py

```diff
@@ -840,6 +840,27 @@ def exists(self):
         except CandidateNotFound:
             return False
 
+    def get_default_group_names(self):
```

Comment:
**REQUIRED**
This method should be a property and probably be a part of `GroupCollection`. Until it's not implemented you can just put this into the tests.